.gitignore

```diff
@@ -52,6 +52,7 @@ $RECYCLE.BIN/
 /test/version_tmp/
 /tmp/
 /spec/tmp
+gem_graph.png
```

Comment:
I committed this file by accident earlier. We should def make sure it's ignored.